packages/amplify-graphql-transformer-migrator/src/schema-migrator.ts

```diff
@@ -0,0 +1,193 @@
+import * as fs from 'fs-extra';
+import { Kind, parse, print, visit } from 'graphql';
+import { migrateKeys } from './migrators/key'
+import { migrateAuth } from './migrators/auth'
+import { migrateConnection } from './migrators/connection'
+import {
+  backupSchema, combineSchemas, DiffDocument,
+  getDefaultAuthFromContext, getSchemaDiffs,
+  readSchemaDocuments,
+  readSingleFileSchema,
+  replaceFile,
+  SchemaDocument,
+  undoAllSchemaMigration,
+} from './utils';
+import { DocumentNode } from 'graphql/language';
+import { prompter, printer } from 'amplify-prompts';
+import path from 'path';
+import { $TSContext, exitOnNextTick } from 'amplify-cli-core';
+import { detectCustomResolvers, detectOverriddenResolvers, detectUnsupportedDirectives, graphQLUsingSQL } from './schema-inspector';
+import { validateModelSchema, SchemaValidationError } from '@aws-amplify/graphql-transformer-core';
+import { updateTransformerVersion } from './state-migrator';
+import { GRAPHQL_DIRECTIVES_SCHEMA } from './constants/graphql-directives';
+
+const cliToMigratorAuthMap: Map<string, string> = new Map<string, string>([
+  ['API_KEY', 'apiKey'],
+  ['AWS_IAM', 'iam'],
+  ['AMAZON_COGNITO_USER_POOLS', 'userPools'],
+  ['OPENID_CONNECT', 'oidc']
+]);
+
+const MIGRATION_URL = "<insert migration docs URL here>";
```

Comment:
Just pointing out that we need a real value for this.

Reply:
Yeah this is going to be a banner message, I'll get it fixed before the actual ship